doc/faq/howto_faq.rst

```diff
@@ -106,13 +106,12 @@ Save transparent figures
 The :meth:`~matplotlib.pyplot.savefig` command has a keyword argument
 *transparent* which, if 'True', will make the figure and axes
 backgrounds transparent when saving, but will not affect the displayed
-image on the screen.
+the image on the screen.
 
-If you need finer grained control, e.g., you do not want full transparency
+If you need finer-grained control, e.g., you do not want full transparency
 or you want to affect the screen displayed version as well, you can set
-the alpha properties directly.  The figure has a
-:class:`~matplotlib.patches.Rectangle` instance called *patch*
-and the axes has a Rectangle instance called *patch*.  You can set
+the alpha properties directly.  The figure has a:class:`~matplotlib.patches.Rectangle` instance called *patch*
+and the axes have a Rectangle instance called *patch*.  You can set
```

Comment:
This might be a result of confusion around MPL's use of the word "axes". In this case, it is ambiguous whether we mean it in the singular sense (as in, axes instance), or the plural sense (multiple axes objects).